### ISSUE TITLE
Archinstall doesn't redirect /etc/resolv.conf automaticaly

### ISSUE THREAD

**Issue by Blurripy** [archlinux/archinstall#1375]
You probably forgot to redirect your resolv.conf, you just need to do this command sudo ln -sf /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf

**Comment by JoelLarson** [comment]
I don't believe it is required for the install to work. It looks recommended when using systemd-resolved as it will use 127.0.0.1:53 (provided by systemd-resolved), so that seems like something to investigate.

More info: https://wiki.archlinux.org/title/Systemd-resolved

**Comment by Blurripy** [comment]
It can be a problem on some apps like Steam

**Comment by ThingsWillCalmUp** [comment]
Chiming in, I know this is a pretty old issue, but I'm running into the same problem.
I'm using "iso" for the network mode, and custom-commands can't make changes to this file by default, causing the build to fail with "device or resource busy".

The issue here is I can't have a consistent search domain without editing this file, and I can't edit the file from archinstall. This is a problem, because archinstall leaves the file in place from the iso. It's not a link, like it should be, to resolved's stub. If I boot into the installed OS, delete /etc/resolv.conf, and reboot, resolved creates the link and things work as they should.

I'm not sure what the correct fix /inside archinstall/ here is. Allow this file to be properly linked to the stub IS required for the install to work properly (dig, for instance, reads resolv.conf), and shouldn't be obscured and unable to be changed/overwritten from the automated install process. At the very least, archinstall should not inhibit the normal operation of resolved. Yes, it defaults to the right local address for resolved, but it also includes a useless "search ." line, which ruins `dig +search <nonqualifiedlocaldomainname>`

As a workaround, consider the following line in custom-commands[] to run at the end of the build. I feel like unmounting should happen automatically PRIOR to the custom commands stage maybe?
`"umount /etc/resolv.conf && rm /etc/resolv.conf"`
